great-idea-website/css/index.css

```diff
@@ -63,4 +63,70 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+.logo {
+    display: inline-block;
+    margin-left: 100px;
+}
+.container {
+    width: 1000px;
+    margin: 0 auto;
+}
+nav{
+    display: inline;
+}
+nav a{
+    text-decoration: none;
+     color: aqua;
+}
+
+.demand{
+    display: inline-block;
+    font-weight: bold;
+    margin: 0px 100px 70px 0px;
+}
+h1 {
+    font-size: 90px;
+    display: block;
+}
+.demand button{
+    display: block;
+    font-size: 20px;
+    margin-left: 60px;
+    background-color: white;
+    padding: 5px 50px;
+
+}
+.features {
```

Comment:
The text sections and image seem to be leaning toward the left. Targeting the `section` element which encloses these text parts and modifying the width might help here.